explainer/effect-generic-trait-declarations.md

```diff
@@ -0,0 +1,562 @@
+- Feature Name: `effect-generic-trait-decls`
+- Start Date: (2024-01-01)
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
```

Comment:
I am guessing this links are supposed to be dead? But it was fairly hard to find this repo so you might want to include a link to here?

(I had to stalk yosh's Github profile)

Reply:
Yeah, they are supposed to be. We could add more links though, if you think that would be helpful? I don't really know of much precedent for checked in draft RFCs like these.

Reply:
I was mostly blind and couldn't find the Github icon in the top right of the mdbook... I think it is fine? Maybe add an explicit link at the start of the mdbook linking back to this repo?